tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,147 @@
+
+raw = mne.io.RawFIF(osp.join(data_path, 'MEG', 'sample',
+                             'sample_audvis_raw.fif'))
+events = mne.find_events(raw)
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2}
+epochs = mne.Epochs(raw, events, event_id, tmin=evoked_ff.times[0],
+                    tmax=evoked_ff.times[-1], baseline=(None, 0),
+                    preload=True)
+evoked = epochs.average()
+
+labels = mne.read_labels_from_annot('sample', subjects_dir=subject_dir)
+label_names = [l.name for l in labels]
+nlabs = len(labels)
+
+###############################################################################
+# Estimate noise covariance from baseline
+cov = mne.compute_covariance(epochs, tmin=None, tmax=0.)
+
+###############################################################################
+# Simulate known signals at two regions
+
+# Generate a known signal to corrupt with point spread
+idx = label_names.index('inferiorparietal-lh')
+signal = np.zeros((nlabs, T))
+signal[idx, :] = 1e-7 * np.sin(5 * 2 * np.pi * times)
+idx = label_names.index('rostralmiddlefrontal-lh')
+signal[idx, :] = 1e-7 * np.sin(7 * 2 * np.pi * times)
+
+###############################################################################
+# Create mask for each label so that the signal is only set at the center
+# vertex.
+hemi_to_ind = {'lh': 0, 'rh': 1}
+for i, label in enumerate(labels):
+    # Labels need values to use center_of_mass function
+    labels[i].values.fill(1.)
+    surf_vertices = fwd['src'][hemi_to_ind[label.hemi]]['vertno']
+    restrict_verts = np.intersect1d(surf_vertices, label.vertices)
+    com = labels[i].center_of_mass(subject='sample',
+                                   subjects_dir=subject_dir,
+                                   restrict_vertices=restrict_verts,
+                                   surf='white')
+    # Get center of vertex index in Label's vertix list
+    cent_idx = np.where(label.vertices == com)[0][0]
+
+    # Mask out all vertices except the center.
+    labels[i].values.fill(0.)
+    labels[i].values[cent_idx] = 1.
+
+###############################################################################
+# Create SourceEstimate object with known signal at center vertices of two
+# labels.
+stc_gen = simulate_stc(fwd['src'], labels, signal, times[0],
+                       times[1] - times[0], value_fun=lambda x: x)
+
+# Use forward solution to generate sensor space data
+evoked_gen = simulate_evoked(fwd, stc_gen, evoked.info, cov, noise_snr,
+                             tmin=0., tmax=1., random_state=seed)
+evoked_gen.info['bads'] = [b for b in inv_op['info']['bads']
+                           if b in evoked.info['ch_names']]
+
+# Apply inverse to project sensor space signal back into source space
+stc_inv = apply_inverse(evoked_gen, inv_op, lambda2, method=method)
+
+###############################################################################
+# Plot original signals
+brain_gen = stc_gen.copy().crop(0.05, None)
```

Comment:
Honestly, this is a piece of code that @Eric89GXL sent me recently and it's been working well so I used it. If what you suggested is more reasonable for the tutorial I'm ok changing it.